EDDiscovery/EDDiscoveryForm.cs

```diff
@@ -139,6 +139,12 @@ private void Form1_Load(object sender, EventArgs e)
 
                 textBoxEDSMApiKey.Text = db.GetSettingString("EDSMApiKey", "");
                 checkBox_Distances.Checked = eddConfig.UseDistances;
+                checkBoxEDSMLog.Checked = eddConfig.EDSMLog;
+
+                checkboxSkipSlowUpdates.Checked = eddConfig.CanSkipSlowUpdates;
+#if DEBUG
+                checkboxSkipSlowUpdates.Visible = true;
+#endif
```

Comment:
I can't think of a good reason for normal users to skip the EDSM loading process, so I've hidden the checkbox from them.